cmd/fdroidcl/search.go

```diff
@@ -24,6 +25,7 @@ var (
 	quiet     = cmdSearch.Flag.Bool("q", false, "Print package names only")
 	installed = cmdSearch.Flag.Bool("i", false, "Filter installed apps")
 	updates   = cmdSearch.Flag.Bool("u", false, "Filter apps with updates")
+	days      = cmdSearch.Flag.Int("d", 0, "Filter apps last updated in the last <n> days")
```

Comment:
I think adding something for negative numbers is necessary. If it turns out that `fdroidcl -h` is ugly or doesn't print well with it, that will have to be solved separately.